src/_locales/nl/messages.json

```diff
@@ -64,7 +64,7 @@
     "description": "Menu entry to view Pomodoro history/stats. Shown in the dropdown menu."
   },
   "timer_paused": {
-    "message": "Timer gepauseerd",
+    "message": "Timer gepauzeeerd",
```

Comment:
Should this be "Timer gepauzeerd"? I definitely don't speak Dutch, but looking online, it seems so.

Reply:
Yes, fixed a typo with a typo, apologies!